### ISSUE TITLE
Gerbera 1.9.2 won't start after Opensuse 15.3 to 15.4 upgrade

### ISSUE THREAD

**Issue by geno11x11** [gerbera/gerbera#2774]
After upgrading Opensuse 15.3 to 15.4 gerbera 1.9.2 refuses to start using `systemctl start gerbera`.  

From online research I've learned that status=127 means the command is not found, however both /usr/bin/gerbera and /etc/gerbera/config.xml are in their proper locations per the path. Not sure where to go from here.  Yes, I could uninstall/reinstall, but this may be a learning opportunity. An explanation on top of the solution would be appreciated. 

**Error message from terminal:**
```
server:/home/geno # systemctl start gerbera
server:/home/ggerbera -c /etc/gerbera/config.xml
gerbera: symbol lookup error: gerbera: undefined symbol: _ZN6spdlog7details7log_msgC1ENS_10source_locEN3fmt2v717basic_string_viewIcEENS_5level10level_enumES6_
server:/etc/gerbera # 
eno # systemctl status gerbera

● gerbera.service - Gerbera Media Server
     Loaded: loaded (/usr/lib/systemd/system/gerbera.service; enabled; vendor preset: disabled)
     Active: activating (auto-restart) (Result: exit-code) since Fri 2022-12-30 15:28:14 PST; 2s ago
    Process: 1909 ExecStart=/usr/bin/gerbera -c /etc/gerbera/config.xml (code=exited, status=127)
   Main PID: 1909 (code=exited, status=127)
```
**I also tried creating a fresh config file:**

```
server:/home/geno # gerbera -c /etc/gerbera/config.xml
gerbera: symbol lookup error: gerbera: undefined symbol: _ZN6spdlog7details7log_msgC1ENS_10source_locEN3fmt2v717basic_string_viewIcEENS_5level10level_enumES6_
server:/etc/gerbera # 
```

**Comment by KarlStraussberger** [comment]
This points to missing c++ libraries. Either rebuild gerbera or install correct cpplibs. Anyway, 1.9.2 is already more than a year. There are more recent packages around.

**Comment by geno11x11** [comment]
I installed a multimedia apps repository and gerbera 1.11.0 per https://opensuse.pkgs.org/15.4/multimedia-apps-x86_64/gerbera-1.11.0-lp154.23.47.x86_64.rpm.html - with the same result during startup.

Plz advise more info reference the correct cpplibs - perhaps a link, and any other information a beginner might need.

Thanks,

GK
     On Friday, December 30, 2022, 04:55:32 PM PST, Karlchen ***@***.***> wrote:  
 
 


This points to missing c++ libraries. Either rebuild gerbera or install correct cpplibs. Anyway, 1.9.2 is already more than a year. There are more recent packages around.

**Comment by KarlStraussberger** [comment]
Maybe you try this package https://build.opensuse.org/package/show/multimedia:apps/gerbera

**Comment by geno11x11** [comment]
Nothing found in Yast Package Manager under cpplibs. What actual package name is needed?
     On Friday, December 30, 2022, 04:55:32 PM PST, Karlchen ***@***.***> wrote:  
 
 


This points to missing c++ libraries. Either rebuild gerbera or install correct cpplibs. Anyway, 1.9.2 is already more than a year. There are more recent packages around.

**Comment by KarlStraussberger** [comment]
Your package lists a set of libraries which obviously points to some weird build environment.
![grafik](https://user-images.githubusercontent.com/23654788/210242743-bc958411-31b1-4bfc-9bfd-44b9638bca01.png)

Did you try the OBS packages with gerbera 1.12 already?

**Comment by geno11x11** [comment]
Hello Karl,

The Opensuse software site at https://build.opensuse.org/package/show/multimedia%3Aapps/gerbera finally showed an "experimental" package for gerbera 1.12.0. I initiated the installation, however it was not successful and displayed the following error:

An error occurred while initializing the software repository.
Details:
https-download.opensuse.org-01673ca1: [https-download.opensuse.org-01673ca1|https://download.opensuse.org/repositories/SUSE:/SLE-15-SP1:/GA/pool/] Valid metadata not found at specified URL
History:
- [https-download.opensuse.org-01673ca1|https://download.opensuse.org/repositories/SUSE:/SLE-15-SP1:/GA/pool/] Repository type can't be determined.


At the end of installation, the following error was displayed:

Software installation
The installation has failed. For more information, see the log file at /var/log/YaST2/y2log. Failure stage was: Adding Repositories
Error Message
An error occurred while attempting to subscribe to the required repositories. Review the yast2 logs for more information.
The following repositories could not be added
 • https://download.opensuse.org/repositories/SUSE:/SLE-15-SP1:/GA/pool/ 
 • https://download.opensuse.org/repositories/SUSE:/SLE-15-SP1:/Update/pool/ 
 • https://download.opensuse.org/repositories/SUSE:/SLE-15-SP2:/GA/pool/ 
 • https://download.opensuse.org/repositories/SUSE:/SLE-15-SP2:/Update/pool/ 
 • https://download.opensuse.org/repositories/SUSE:/SLE-15-SP3:/GA/pool/ 
 • https://download.opensuse.org/repositories/SUSE:/SLE-15-SP4:/GA/pool/ 
 • https://download.opensuse.org/repositories/SUSE:/SLE-15:/Update/pool/

My guess is that the links are broken. An online search resulted in the following message:

Resource is no longer available!
The requested URL is no longer available on this server and there is no forwarding address. If you followed a link from a foreign page, please contact the author of this page.

If you think this is a server error, please contact the webmaster.

Error 404

Suggestions?


     On Monday, January 2, 2023, 06:13:17 AM PST, Karlchen ***@***.***> wrote:  
 
 


Your package lists a set of libraries which obviously points to some weird build environment.


Did you try the OBS packages with gerbera 1.12 already?

**Comment by paolostivanin** [comment]
How did you install gerbera in the first place?
Gerbera is officially available only for Tumbleweed. 
If you want it on Leap, you've gotta use `opi` and select the one from `multimedia:apps`(or add directly the whole repository, but I'd advise against that).

To me this error looks like something on your end, like mixing versions/missing recompile/etc.

**Comment by geno11x11** [comment]
Still working on this problem. Is Gerbera support for LEAP discontinued? It has been provided for years but it seems to have disappeared from the OpenSUSE LEAP software download page.

I was able to locate a 3rd party downgraded gerbera version 1.11 for Opensuse 15.4 at packages.org and gave it a try. Multiple errors arose, so I uninstalled Gerbera, removed the repository and started over. This time installation  completed without errors but systemd still refuses to start with a (code=exited, status=127) error.  I tried rebuilding the config file and I got the following error repeatedly:

```
gerbera -c /etc/gerbera/config.xml
gerbera: symbol lookup error: gerbera: undefined symbol: _ZN6TagLib3Ogg4Opus4File11isSupportedEPNS_8IOStreamE
```

Not sure what to do about it. Suggestions? Pls include explanations & examples.
